### PR TITLE
`cirq-superstaq` documentation for `Cirq` repository

### PR DESCRIPTION
**Pull request by dowusu-antwi**
Provides documentation on how to access `cirq-superstaq` and an example / tutorial, to potentially be included in [Cirq's documentation](https://github.com/quantumlib/Cirq/tree/master/docs).

If included in `Cirq`, `access.md` should be at `Cirq/docs/cirq-superstaq/` and `getting_started_cirq_superstaq` should be at `Cirq/docs/tutorials/cirq-superstaq/`, i.e., the file directory structure would be as follows:

cirq-docs-tree

Fixes #87.

### CODE REVIEW

File: cirq_docs/getting_started_cirq_superstaq.ipynb [hunk 1]
```diff
@@ -0,0 +1,189 @@
+{
```
Comment (vtomole) [inline cirq_docs/getting_started_cirq_superstaq.ipynb:1]: <p>Typo</p><p> </p><blockquote>Build a cirquit</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/cirq-superstaq/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

[the remark this answers is not in the document]
Comment (dowusu-antwi) [reply cirq_docs/getting_started_cirq_superstaq.ipynb:1]: Fixed.

**Comment by vtomole** [comment]
> Also, should we include instructions about how to obtain an IBMQ token and how to get cirq_superstaq.Service to work with both tokens?

Yes, but on a different notebook.

File: cirq_docs/access.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Setting up access and authentication to Cirq-SuperstaQ
+
+Cirq-SuperstaQ is used to access [SuperStaq](https://www.super.tech/about-superstaq/) via a Web API through [Cirq](https://github.com/quantumlib/Cirq). Cirq programmers can take advantage of the applications, pulse level optimizations, and write-once-target-all features of SuperstaQ with this package.
```
Comment (richrines1) [inline cirq_docs/access.md:3]:
```suggestion
Cirq-SuperstaQ is used to access [SuperstaQ](https://www.super.tech/about-superstaq/) via a Web API through [Cirq](https://github.com/quantumlib/Cirq). Cirq programmers can take advantage of the applications, pulse level optimizations, and write-once-target-all features of SuperstaQ with this package.
```

File: docs/access.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Please note that Python version `3.7` or higher is required.
+
+## Accessing Cirq-SuperstaQ
+
+Creating and submitting a circuit through Cirq-SuperstaQ requires an API access token obtained at https://superstaq.super.tech.
+
+## Next steps
+
+[Getting started with Cirq-SuperstaQ](../tutorials/cirq-superstaq/getting_started_cirq_superstaq.ipynb)
```
Comment (vtomole) [inline docs/access.md:13]: Also link to the IBM notebook,

File: docs/getting_started_cirq_superstaq.ipynb [hunk 4]
```diff
@@ -0,0 +1,212 @@
+{
```
Comment (vtomole) [inline docs/getting_started_cirq_superstaq.ipynb:1]: <p>Nit: Would commas be better here?</p><p> </p><blockquote>SuperstaQ offers access to multiple target backends: <a href="https://aqt.lbl.gov/" target="_blank">AQT</a>, <a href="https://www.ibm.com/quantum-computing/" target="_blank">IBM Q</a>, <a href="https://ionq.com/" target="_blank">IonQ</a> and <a href="https://www.rigetti.com/" target="_blank">Rigett</a>. Also I don't think Azure gives access to QCI. Please confirm.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/cirq-superstaq/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Pull request merged**